### PR TITLE
feat(core):handle oneof validation

### PR DESCRIPTION
**Pull request by jremy42**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request.
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Release note for [CHANGELOG](https://github.com/scaleway/scaleway-cli/blob/master/CHANGELOG.md):

```release-note

```

### CODE REVIEW

File: internal/core/validate_test.go [hunk 1]
```diff
@@ -371,3 +371,164 @@ func Test_ValidateDeprecated(t *testing.T) {
 		),
 	}))
 }
+
```
Comment (remyleone) [inline internal/core/validate_test.go:374]: Add tests for validateRequiredArgs(

File: cmd/scw/testdata/test-all-usage-baremetal-server-install-usage.golden [hunk 2]
```diff
@@ -13,7 +13,7 @@ ARGS:
   server-id             Server ID to install
   os-id                 ID of the OS to installation on the server
   hostname              Hostname of the server
-  [all-ssh-keys]        Add all SSH keys on your baremetal instance (cannot be used with ssh-key-ids)
+  all-ssh-keys          Add all SSH keys on your baremetal instance (cannot be used with ssh-key-ids)
```
Comment (jremy42) [inline cmd/scw/testdata/test-all-usage-baremetal-server-install-usage.golden:16]:
```suggestion
  [all-ssh-keys]          Add all SSH keys on your baremetal instance (cannot be used with ssh-key-ids)
```

File: cmd/scw/testdata/test-all-usage-baremetal-server-install-usage.golden [hunk 3]
```diff
@@ -13,7 +13,7 @@ ARGS:
   server-id             Server ID to install
   os-id                 ID of the OS to installation on the server
   hostname              Hostname of the server
-  [all-ssh-keys]        Add all SSH keys on your baremetal instance (cannot be used with ssh-key-ids)
+  [all-ssh-keys]          Add all SSH keys on your baremetal instance (cannot be used with ssh-key-ids)
```
Comment (jremy42) [inline cmd/scw/testdata/test-all-usage-baremetal-server-install-usage.golden:16]:
```suggestion
  [all-ssh-keys]         Add all SSH keys on your baremetal instance (cannot be used with ssh-key-ids)
```

File: cmd/scw/testdata/test-all-usage-baremetal-server-install-usage.golden [hunk 4]
```diff
@@ -13,7 +13,7 @@ ARGS:
   server-id             Server ID to install
   os-id                 ID of the OS to installation on the server
   hostname              Hostname of the server
-  [all-ssh-keys]        Add all SSH keys on your baremetal instance (cannot be used with ssh-key-ids)
+  [all-ssh-keys]         Add all SSH keys on your baremetal instance (cannot be used with ssh-key-ids)
```
Comment (jremy42) [inline cmd/scw/testdata/test-all-usage-baremetal-server-install-usage.golden:16]:
```suggestion
  [all-ssh-keys]        Add all SSH keys on your baremetal instance (cannot be used with ssh-key-ids)
```

**Review by remyleone — APPROVED** [review]

**Pull request merged**